client/src/pages/Favourites.js

```diff
@@ -15,40 +16,24 @@ const FavouriteDiv = styled.div`
 `;
 
 const Favourite = () => {
+  const { toggleFavorite, favorites } = useFavorites("favorites", []);
+
+  const storagefavorites = JSON.parse(localStorage.getItem("favorites"));
+  console.log(favorites);
+
   return (
     <FavouriteDiv>
-      <PageHeadline>Suche</PageHeadline>
+      <PageHeadline>Favoriten</PageHeadline>
       <Standort>
-        <li details>
-          Suchergebnis
-          <FavouriteButton>
-            <img src={FavoriteOffpath} alt="favorite_off" />,
-          </FavouriteButton>
-        </li>
-        <li>
-          Suchergebnis
-          <FavouriteButton>
-            <img src={FavoriteOnpath} alt="favorite_on" />
-          </FavouriteButton>
-        </li>{" "}
-        <li>
-          Suchergebnis
-          <FavouriteButton>
-            <img src={FavoriteOnpath} alt="favorite_on" />
-          </FavouriteButton>{" "}
-        </li>{" "}
-        <li>
-          Suchergebnis
-          <FavouriteButton>
-            <img src={FavoriteOnpath} alt="favorite_on" />
-          </FavouriteButton>{" "}
-        </li>
-        <li>
-          Suchergebnis
-          <FavouriteButton>
-            <img src={FavoriteOnpath} alt="favorite_on" />
-          </FavouriteButton>{" "}
-        </li>
+        {storagefavorites.map((name) => (
+          <li key={name}>
+            <Link to={`/Location/${name}`}>{name}</Link>
```

Comment:
Routes are usually written in lower case and you target multiple locations
```suggestion
            <Link to={`/locations/${name}`}>{name}</Link>
```

Reply:
Changed the casing on the routes. Not sure why "location" should be "locations" (with an "s") because I am only ever targeting one particular location.